scripts/utils/constructProposalCalldata.ts

```diff
@@ -59,12 +64,18 @@ function getDAOCalldata(proposal: ExtendedAlphaProposal): string {
   return calldata;
 }
 
-function getTimelockCalldata(proposal: ExtendedAlphaProposal, proposalInfo: TemplatedProposalDescription): string {
+function getTimelockCalldata(
+  proposal: ExtendedAlphaProposal,
+  proposalInfo: TemplatedProposalDescription,
+  podConfig?: PodConfig
+): string {
   const proposeFuncFrag = new Interface([
     'function scheduleBatch(address[] calldata targets,uint256[] calldata values,bytes[] calldata data,bytes32 predecessor,bytes32 salt,uint256 delay) public',
     'function executeBatch(address[] calldata targets,uint256[] calldata values,bytes[] calldata data,bytes32 predecessor,bytes32 salt) public'
   ]);
 
+  const metadataFuncFrag = new Interface(['function registerProposal(uint256,uint256,string) external']);
```

Comment:
Would be better to import these function fragments from a compiled ABI than to inline them

Reply:
Yeah sure